docs/toc.md

```diff
@@ -13,7 +13,10 @@
             - [5. Get playing]({{urlRoot}}/projects/fps/get-started/get-playing)
             - [6. View your game world]({{urlRoot}}/projects/fps/get-started/view-game-world)
         - [Health packs tutorial]({{urlRoot}}/projects/fps/tutorial)
-    - [Blank Starter Project]({{urlRoot}}/projects/blank/overview)
+    - Blank Starter Project
+        - [Overview]({{urlRoot}}/projects/blank/overview)
+        - Setup
```

Comment:
Do we want this to have a sub-topic for just Add schema?

Reply:
I think it's fine for now - as the `add schema` bit will be covered by the blank project tutorial

(also a user _should_ know how to add schema if they take a look at the pickups tutorial)